mentorship-code-of-conduct.html

```diff
@@ -6,6 +6,56 @@
 <div class="page-header">
     <h1>Mentorship Code of Conduct</h1>
 </div>
-<div class="container">
-    TODO
+<div class="container  code-of-conduct">
+    <div id="mentee-section" class="col-12 section">
+        <h2>Code Of Conduct for Mentees</h2>
+        <ol>
+            <li><em>Women Coding Community is an inclusive community</em>, dedicated to providing an empowering experience for everyone who participates in or supports our community, regardless of gender, gender identity and expression, sexual orientation, ability, physical appearance, body size, race, ethnicity, age, religion, socioeconomic status, caste, creed, political affiliation, or preferred programming language(s).</li>
+            <li>Cancel or reschedule appointments with a minimum of 24 hours prior notice by providing a reason using preferable channel (email, slack, or anything else you both chose as the channel of communication) to the mentor.</li>
+            <li>No-shows are unacceptable for mentors and mentees. No-shows should be informed to the Mentorship Programme Team by <a href="https://bit.ly/women_coding_community">Slack </a>, channel #mentorship. The Mentorship Programme Team will evaluate case by case, and this is one of the possible actions that could result in exclusion from the current and future mentorship program cycles.</li>
+            <li>No-replies by email or slack to the Mentorship Programme Team and/or mentor in a week are unacceptable during the program. Participants who applied but failed to reply without any reason provided will be banned from the current cycle.</li>
+            <li>Create a <a href="https://bit.ly/women_coding_community">Slack Account</a>, join the #mentorship channel and check slack messages in the program to remain updated with all communication.</li>
```

Comment:
add 'target="_blank" for external links to have those links open in a new window